### PR TITLE
chose a random, 6 character long hostname to avoid conflict with multiple platforms in the same network

### PR DESCRIPTION
**Pull request by paukul**

### CODE REVIEW

**Comment by kdomanski** [comment]
@hinnerk LGTM
The rationale behind this was the domain collision when multiple platforms were being set up on a single network.

**Comment by hinnerk** [comment]
Please set it only if it's not already set, so that a host name can be chosen on installation and follow the naming convention for install config variables (PLATFORM_INSTALL_HOSTNAME).

**Comment by paukul** [comment]
@hinnerk the script only sets it if the hostname file doesn't exist. Sour works without the need of
Any external configuration parameter. Or what am I missing?

File: platform-configure.sh [hunk 1]
```diff
@@ -167,6 +168,14 @@ function install_platform() {
   $DOCKER kill $CONTAINER_NAME 2>/dev/null || true
   $DOCKER rm $CONTAINER_NAME 2>/dev/null || true
 
+  mkdir -p /etc/protonet
+  [[ -d /etc/protonet/hostname ]] && rm -rf /etc/protonet/hostname
+  if [[ ! -f /etc/protonet/hostname ]]; then
```
Comment (paukul) [inline platform-configure.sh:173]: @hinnerk the script checks for the `/etc/protonet/hostname` file here

**Comment by hinnerk** [comment]
With 03c308e11acedb78fac69a5694e7e4bf9153168e it's now possible to set the hostname on installation. Please test and then merge.

**Pull request merged**